classes/article/ArticleGalleyDAO.inc.php

```diff
@@ -307,7 +361,9 @@ function deleteById($galleyId, $articleId = null) {
 			$params
 		);
 		if ($this->getAffectedRows()) {
-			$this->update('DELETE FROM submission_galley_settings WHERE galley_id = ?', array((int) $galleyId));
+			$this->update('
+				DELETE FROM submission_galley_settings
+				WHERE galley_id = ? AND version = ?', array((int) $galleyId, (int)$submissionRevision));
```

Comment:
should also the new table submission_galley_files be considered when deleting a galley?